errors.go

```diff
@@ -304,4 +304,6 @@ var (
 	ErrInvalidAsciiArmor = errors.New("ascii armor is invalid")
 
 	ErrRequiredNamespace = errors.New("namespace is required for public registry")
+
+	ErrVersionValidForPlanOnly = errors.New("configuring a run specific terraform version is only valid for plan-only runs")
```

Comment:
I think we should be explicit here

```suggestion
	ErrTerraformVersionValidForPlanOnly = errors.New("setting terraform-version is only valid when plan-only is set to true")
```